.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,7 +6,7 @@
 
 <!-- Before submitting a pull request, please make sure you have answered the following: -->
 
-- [ ] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
+- [ ] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
```

Comment:
thanks for fixing this - do you mind doing this in a separate PR though?

Reply:
This has already been added in https://github.com/shapeshift/web/pull/2650/files#diff-18813c86948efc57e661623d7ba48ff94325c9b5421ec9177f724922dd553a35R9

@southbears just merged develop from GitHub UI - this will make it easier to reviewers to get this one to the finish line